### DISCUSSION TITLE
AWS Credentials

### DISCUSSION THREAD

**Discussion by harishsarmav** [learndevopsonline/learndevopsonline#749] [Q&A]
![image](https://user-images.githubusercontent.com/101693424/235366346-c8d8cf83-09f1-4fc1-b014-ddd61d07f961.png)

Hi @r-devops , im facing above issue.. credentials updated even though im facing this error sir.. please help me to resolve this issue

**Comment by Arafique458** [comment]
Hi @harishsarmav I believe we were suppose to create the role for the EC2 instance (workstation) server and assign the role to the workstation. we were not suppose to hard code the credentials in the workstation. 

I hope this helps.

**Reply by harishsarmav** [reply]
thanks for your reply. Yes I created it.. i think there is some other issue

**Reply by Arafique458** [reply]
Also did you assign the role to the workstation as well.

<img width="781" alt="image" src="https://user-images.githubusercontent.com/120502690/235392121-7895209c-3e4b-493f-acfa-b8d9c6ef8a63.png">

**Reply by Arafique458** [reply]
Are you working on Terraform or on the Shell-project section?

**Reply by harishsarmav** [reply]
i figured it out. thanks once agiain
